rust/src/common.rs

```diff
@@ -14,6 +14,24 @@ macro_rules! take_until_and_consume (
   );
 );
 
+
+#[cfg(not(feature = "debug-validate"))]
+#[macro_export]
+macro_rules! debug_validate_bug_on (
+  ($item:expr) => {};
+);
+
+
+#[cfg(feature = "debug-validate")]
+#[macro_export]
+macro_rules! debug_validate_bug_on (
+  ($item:expr) => {
+    if $item {
+        panic!("Value should not be NULL!");
```

Comment:
curious: can we just feed $item to `panic!` so its printed in the backtrace?

Reply:
`$item` is just a bool value. So true or false would be printed in its place. Do we want that? Panic would probably tell us where it occurred `thread 'dcerpc::dcerpc::tests::test_handle_bindctxitem' panicked at 'Value should not be NULL! true', src/dcerpc/dcerpc.rs:304:9`
What do you think?

Reply:
I'm fine with this (+ requested change from newer PR)